server/controllers/class/getAllStudentHowSubmitTasks.ts

```diff
@@ -0,0 +1,14 @@
+import { Request, Response, NextFunction } from 'express';
+import { getAllStudentWhoSubmitTasksQuery } from '../../queries/class';
+
+const getAllStudentWhoSubmitTasks = async (req:Request, res:Response, next:NextFunction) => {
+  try {
+    const { assignmentId } = req.params;
+    const data:any = await getAllStudentWhoSubmitTasksQuery(assignmentId);
```

Comment:
This controller should get both, those who submit the task and who do not, you can check your task in the HackMD.